servers/physics_2d/physics_2d_server_wrap_mt.cpp

```diff
@@ -84,11 +84,11 @@ void Physics2DServerWrapMT::step(real_t p_step) {
 
 void Physics2DServerWrapMT::sync() {
 
-	if (step_sem) {
+	if (thread) {
```

Comment:
@reduz, please validate this change.

For the semaphore to be "in-object", now it's created unconditionally, so this check couldn't be based in `step_sem` anymore.

The new logic is equivalent, isn't it?

Reply:
its fine

Reply:
Comment from @reduz:
```
i think its ok, the wrapmt classes should probably be rewritten using spinlocks mainly
```